Manual/manJHUGenerator.tex

```diff
@@ -28,7 +28,7 @@
 \begin{center}
 \small
 For simulation, reweighting, and analysis of single-produced resonances at hadron colliders \\
-(version v7.5.0, release date July 10, 2021) \\
+(version v7.5.1, release date January 21, 2022) \\
```

Comment:
Please put the expected release date so that we don't have to make yet one more PR to change it again. Monday maybe?